pkg/options/options.go

```
@@ -232,3 +232,19 @@ func validateFlag(name, value string) error {
+
+// stringFlag registers a string flag if it does not already exist on the
+// default flag set, otherwise it returns a pointer to the existing flag's
+// default value. This prevents "flag redefined" panics when another package
+// (e.g. controller-runtime via envtest) registers the same flag name in an init().
+//
+// The alternative is to use a custom build tag which has
+// some disadvantages as the IDE must be aware of the build tag and the test must be run with the build tag.
+func stringFlag(name, value, usage string) *string {
+	if f := flag.Lookup(name); f != nil {
+		v := f.DefValue
+		return &v
```

golangci-lint (2.12.2)

[error] return with no blank line before

(nlreturn)